src/components/paypalMeDataPropTypes.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+
+export default PropTypes.shape({
+    /** This is always 'PayPal.me' */
+    title: PropTypes.string,
+
+    /** The paypalMe address */
+    description: PropTypes.string,
+
```

Comment:
Did we name it this way so that it was in the same format as the other bank / cards?

Reply:
+1